### ISSUE TITLE
Display toast when edit is saved

### ISSUE THREAD

**Issue by eliserichards** [mozilla-lockwise/lockwise-android#819]
Depends on #628 

## Requirements
* Should display toast (on item detail) to confirm that an edit is saved

## Designs
* TBD

**Comment by changecourse** [comment]
Removing this from MVP functionality for 1.3.0.

Updated the functional requirements in #821 to transition from the create view to the read-only view of the login detail, removing the need for a confirmation toast at this time.

We can revisit if we feel this is necessary in the future. (After performing a competitive analysis of the feature, no one displays toast confirmations, it is implied by the routing to the read-only state)

**Comment by eliserichards** [comment]
Do we think this is still applicable for Skyline? @sandysage @changecourse

**Comment by changecourse** [comment]
Closed as duplicate of #1009
